### PR TITLE
The last dance

### PR DESCRIPTION
**Pull request by perryr16**
several small fixes found while doublechecking work

### CODE REVIEW

File: app/controllers/apps_controller.rb [hunk 1]
```diff
@@ -18,40 +18,43 @@ def show
...
-    
+
     if @app.save
       params[:pet_ids].each do |id|
         PetApp.create(pet_id: id, app_id: @app.id, approved: false)
         session[:favorite].delete(id.to_s)
       end
-
-      flash[:notice] = "Your application for the selected pets has been submitted"
+      flash[:notice] = "Your application for #{pets_on_app} has been submitted"
```
Comment (perryr16) [inline app/controllers/apps_controller.rb:27]: list each pet on app. see method in private

File: app/controllers/apps_controller.rb [hunk 2]
```diff
@@ -72,4 +75,20 @@ def app_params
                   )
   end
 
+  def empty_params
```
Comment (perryr16) [inline app/controllers/apps_controller.rb:78]: perhaps a cleaner way to do this

File: spec/features/apps/new_app_spec.rb [hunk 3]
```diff
@@ -6,6 +6,7 @@
     @shelter1 = Shelter.create(name: "the lab", address: "666 dog Ave", city: "ruff town", state: "DG", zip: "12345")
     @pet1 = @shelter1.pets.create(name: "Dr. Dog", age: "4", sex: "male", description: "well educated", image: "https://i.redd.it/4ygdq7e6gze11.jpg", adoptable: true)
     @pet2 = @shelter1.pets.create(name: "Oreo", age: "55", sex: "female", description: "best if dipped in milk", image: "https://image.shutterstock.com/image-photo/cute-american-shorthair-cat-kitten-260nw-352176329.jpg", adoptable: false)
+    @pet3 = @shelter1.pets.create(name: "Toad", age: "55", sex: "female", description: "best if dipped in milk", image: "https://image.shutterstock.com/image-photo/cute-american-shorthair-cat-kitten-260nw-352176329.jpg", adoptable: false)
```
Comment (perryr16) [inline spec/features/apps/new_app_spec.rb:9]: added pet for more thurough testing

Comment (muydanny) [reply spec/features/apps/new_app_spec.rb:9]: excellent

**Pull request merged**